src/services/action/utils/export.ts

```diff
@@ -26,7 +26,7 @@ export interface ExportActionsInArchiveOutput {
 }
 
 // faltten object nested keys to have as item.id, member.id to be used for export csv header
-const flattenObject = (obj, prefix = '') => {
+const flattenObject = (obj: object, prefix: string = ''): object => {
```

Comment:
`object` type is no recommended.
Either use a recursive type using the object notation `type RecursiveObject = {[key: string]: string | number | RecursiveObject }` and for the return type specify that it will have only a single level: `type ReturnObject = {[key: string]: string | number}`

Reply:
As we discussed during the standup meeting, maybe it would be better to handle each table specifically instead of treating them generally like that in order to provide a more tailored approach for each data.